<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastWindow</class>
 <widget class="QDialog" name="LastWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>760</y>
     <width>256</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>780</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>去过的城市：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cq">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_bj">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>320</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_xg">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>730</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_sh">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>500</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tj">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>320</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_lz">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>410</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_am">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>720</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_by">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>440</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_dh">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ts">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_sjz">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_zjk">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>340</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gs">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>265</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cq">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>489</y>
     <width>185</width>
     <height>113</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bj">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>275</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hb">
   <property name="geometry">
    <rect>
     <x>793</x>
     <y>238</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hn">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>420</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="zj">
   <property name="geometry">
    <rect>
     <x>878</x>
     <y>490</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ah">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>398</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hlj">
   <property name="geometry">
    <rect>
     <x>859</x>
     <y>-20</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="jl">
   <property name="geometry">
    <rect>
     <x>937</x>
     <y>151</y>
     <width>251</width>
     <height>185</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ln">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>191</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nmg">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>27</y>
     <width>921</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="xj">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>83</y>
     <width>491</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fj">
   <property name="geometry">
    <rect>
     <x>803</x>
     <y>586</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="jx">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>551</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sc">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>455</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="js">
   <property name="geometry">
    <rect>
     <x>823</x>
     <y>426</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sd">
   <property name="geometry">
    <rect>
     <x>797</x>
     <y>321</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sh">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>482</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tj">
   <property name="geometry">
    <rect>
     <x>837</x>
     <y>297</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>hb</zorder>
  <zorder>bj</zorder>
  <zorder>cq</zorder>
  <zorder>gs</zorder>
  <zorder>textBrowser</zorder>
  <zorder>label</zorder>
  <zorder>label_cq</zorder>
  <zorder>label_bj</zorder>
  <zorder>label_xg</zorder>
  <zorder>label_sh</zorder>
  <zorder>label_tj</zorder>
  <zorder>label_am</zorder>
  <zorder>label_by</zorder>
  <zorder>label_dh</zorder>
  <zorder>label_ts</zorder>
  <zorder>label_sjz</zorder>
  <zorder>label_zjk</zorder>
  <zorder>label_lz</zorder>
  <zorder>hn</zorder>
  <zorder>zj</zorder>
  <zorder>ah</zorder>
  <zorder>hlj</zorder>
  <zorder>jl</zorder>
  <zorder>ln</zorder>
  <zorder>nmg</zorder>
  <zorder>xj</zorder>
  <zorder>fj</zorder>
  <zorder>jx</zorder>
  <zorder>sc</zorder>
  <zorder>js</zorder>
  <zorder>sd</zorder>
  <zorder>sh</zorder>
  <zorder>tj</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
